OHHTTPStubsDemo/OHHTTPStubsDemo.xcodeproj/project.pbxproj

```diff
@@ -278,12 +329,17 @@
 		};
 		098FBDE815D704E800623941 /* Debug */ = {
 			isa = XCBuildConfiguration;
+			baseConfigurationReference = B5AD292FAB98FC9C7637896A /* Pods.debug.xcconfig */;
 			buildSettings = {
 				CLANG_ENABLE_OBJC_ARC = YES;
 				GCC_PRECOMPILE_PREFIX_HEADER = YES;
 				GCC_PREFIX_HEADER = "OHHTTPStubsDemo-Prefix.pch";
 				INFOPLIST_FILE = "OHHTTPStubsDemo-Info.plist";
-				IPHONEOS_DEPLOYMENT_TARGET = 4.3;
+				IPHONEOS_DEPLOYMENT_TARGET = 6.0;
```

Comment:
Doesn't this change have other consequences, like raising the minimum iOS version for Carthage users (as Carthage use the settings in the Xcode project building the lib) — even if that's not big a deal as Carthage requires dynamic frameworks so every project using Carthage is probably iOS8+, but just wondering if it may have other side effects.

Note that I won't mind raising the minimum requirement to 6.0 (everything under 5.0 is quite dead anyway those days), but in that case be sure not to forget to bump the pod version up to the next major version to be semver-compliant.